mods/ts/rules/nod-structures.yaml

```diff
@@ -25,6 +25,7 @@ NAPOWR:
 		Range: 4c0
 		MaxHeightDelta: 3
 	WithIdleOverlay@LIGHTS:
+		RequiresCondition: !disabled
```

Comment:
Most other structures have blinking lights or other animations that should also be paused when disabled.  Why stop with just the power plants?